basic_gdeploy_doc.md

```diff
@@ -0,0 +1,298 @@
+
+1.) clone the gdeploy repository by running the below command in your preferred directory.
+
+    [poo@poohostname ~]$ git clone git@github.com:gluster/gdeploy.git
+
+2.) Go to inside gdeploy folder. Like this:
+    
+    $ cd gdeploy/
+
+3.) You should be the root user to run the requirement.txt file. You can become root by running
+    the following command at the command prompt and then entering your password.
+
+    [poo@poohostname gdeploy]$ su
+    Password:
+
+
+4.)  Install the requirements.txt file.
+
+    [root@poohostname gdeploy]# pip install -r requirements.txt
+
+5.) Now we have to setup gdeploy by running below command.
+
+    [root@poohostname gdeploy]# ./gdeploy_setup.sh
+    
+or
+
+You can do the same manually as well.
+
+1.  Add ansible modules to ANSIBLE_LIBRARY environment variable by running the below command.
+
+    echo "export Ansible_LIBRARY=$ANSIBLE_LIBRARY:'/path/to/gdeploy/modules'" >> ~/.bashrc
+
+    In my case, this would be:
+
+        echo "export Ansible_LIBRARY=$ANSIBLE_LIBRARY:'/home/poo/gdeploy/modules'" >> ~/.bashrc
+
+
+    Add ansible playbooks(inside the templates directory) to GDEPLOY_TEMPLATES environment variable.
+    
+
+       echo "export GDEPLOY_TEMPLATES='path/to/gdeploy'" >> ~/.bashrc
+
+    In my case, this would be:
+
+      echo "export GDEPLOY_TEMPLATES='/home/poo/gdeploy'" >> ~/.bashrc
+
+1. Install glusterlib module using setuptool.
+
+    python setup.py install
+
+
+ If you have properly followed all the steps, gdeploy should be installed on your machine. To
+  check this, you can run command gdeploy --version. The output should look like this.
+
+    [poo@poohostname /]$ gdeploy --version
+    gdeploy 2.0
+
+
+
+
+
+##### Installation of gdeploy using  operating system's package manager:
+
+Download gdeploy ( .rpm file for Fedora OS ) from here→ http://download.gluster.org/pub/gluster/gdeploy/2.0/
+
+Prerequisites to install gdeploy:
+
+  * ansible>=1.9  
+  * python>=2.6  
+  * python(abi) = 2.7
+  * python-eventlet
+
+These are the commands that needs to be executed to install the prerequisites on Fedora 23 VM.
+
+      dnf install ansible
+      dnf install python-eventlet
+
+When you installed the above packages, go ahead and install the gdeploy by running the 
+below command from the directory where you downloaded it. You need to be the root user 
+to install gdeploy.
+
+    [root@poo Downloads]# rpm -Uvh gdeploy-2.0-0.fc23.noarch.rpm
+
+
+gdeploy should be installed without fail if you followed the steps properly. In case,
+ if you happen to encounter any error even after following the above steps. 
+Please do contact us here.
+    
+
+
+Now you can use gdeploy tool to configure glusterfs. Before you can use gdeploy to
+configure glusterfs you need to have glusterfs-client installed on the machine from 
+where you’ll be mounting the volume. And glusterfs-server installed on the machine 
+where you’ll be storing your files/data.
```

Comment:
ACK. A quick start is intended for an audience who are familiar with the "GlusterFS way". This reader wants to check how day-to-day operations can be replaced by gdeploy.